cypress/integration/find.spec.js

```diff
@@ -79,27 +78,44 @@ describe('find* dom-testing-library commands', () => {
   })
 
   it('findAllByTestId', () => {
-    cy.findAllByTestId(/^image-with-random-alt-tag-\d$/).should('have.length', 2)
+    cy.findAllByTestId(/^image-with-random-alt-tag-\d$/).should(
+      'have.length',
+      2,
+    )
   })
 
   /* Test the behaviour around these queries */
 
-  it('findByText with should(\'not.exist\')', () => {
+  it("findByText with should('not.exist')", () => {
     cy.findAllByText(/^Button Text \d$/).should('exist')
-    cy.findByText('Non-existing Button Text', {timeout: 100}).should('not.exist')
+    cy.findByText('Non-existing Button Text', {timeout: 100}).should(
+      'not.exist',
+    )
+  })
+
+  it('findByText with a previous subject', () => {
+    cy.get('#nested')
+      .findByText('Button Text 1')
+      .should('not.exist')
+    cy.get('#nested')
+      .findByText('Button Text 2')
+      .should('exist')
   })
 
   it('findByText within', () => {
     cy.get('#nested').within(() => {
-      cy.findByText('Button Text 2').click()
+      cy.findByText('Button Text 1').should('not.exist')
+      cy.findByText('Button Text 2').should('exist')
     })
   })
 
   it('findByText in container', () => {
-    return cy.get('#nested')
-      .then(subject => {
-        cy.findByText(/^Button Text/, {container: subject}).click()
-      })
+    // NOTE: Cypress' `then` doesn't actually return a promise
+    // eslint-disable-next-line jest/valid-expect-in-promise
```

Comment:
We should probably disable the jest plugin for these tests. But we can do that in another PR :)